keyboards/acekeyboard/titan60/keymaps/birgus-special/keymap.c

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2021 keebnewb
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+    SHRUG = SAFE_RANGE,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case SHRUG:
+        if (record->event.pressed) {
+            // when keycode SHRUG is pressed
+            SEND_STRING("¯\\_(ツ)_/¯");
```

Comment:
This will not work, you need to use the Unicode feature instead.

Reply:
@fauxpark I removed this file. It was added by the board runner and I'll come back and submit it as another PR later so I can learn more about how the unicode features of QMK work :)